eahub/templates/eahub/profile.html

```diff
@@ -37,31 +37,22 @@
     <div class="col-xs-12 col-md-6">
       <h1 class="word-break">{{ profile.name }}</h1>
       {% if request.user.id == profile.user_id %}
-        <a href="{% url 'edit_profile' %}">
-            <div class="btn btn-default purple">
-                <span><i class="fa fa-pencil-alt"></i> Edit</span>
-            </div>
-        </a>
-        <a href="{% url 'account_email' %}">
-            <div class="btn btn-default purple">
-                <span><i class="fa fa-pencil-alt"></i> Change Email Address</span>
-            </div>
-        </a>
-        <a href="{% url 'account_change_password' %}">
-            <div class="btn btn-default purple">
-                <span><i class="fa fa-pencil-alt"></i> Change Password</span>
-            </div>
-        </a>
+      <div class="btn btn-default purple">
+        <span><a href="{% url 'edit_profile' %}"><i class="fa fa-pencil-alt"></i> Edit</a></span>
+      </div>
```

Comment:
@mattg95, @sebgrebe, we must be highly careful here - this is a button 20% of which isn't clickable. I fixed them thought the site one by one before, but now we have to prevent those occurrences in the future

Reply:
Thanks for catching this! This wasn't intentional, the branch probably hadn't been rebased for a while.